subsys/bluetooth/controller/Kconfig

```diff
@@ -101,6 +101,9 @@ config BT_CTLR_READ_ISO_LINK_QUALITY_SUPPORT
 		   BT_CTLR_PERIPHERAL_ISO_SUPPORT
 	bool
 
+config BT_CTLR_LE_POWER_CONTROL_SUPPORT
```

Comment:
what is the difference to CONFIG_BT_CTLR_LE_POWER_CONTROL used in other places?

Reply:
Remnant of the beginning, this file is used by the zephyr controller, so will remove this.